internal/worker/dbaccessor/doc.go

```diff
@@ -1,51 +1,76 @@
-// Copyright 2023 Canonical Ltd.
-// Licensed under the AGPLv3, see LICENCE file for details.
+// Copyright 2023 Canonical Ltd. Licensed under the AGPLv3, see LICENCE file for
+// details.
 
-// Package dbaccessor is a worker that provides access to the SQLite database
-// via the Dqlite cluster. It is responsible for coordinating the Dqlite
-// cluster, managing the lifecycle of the database connections, and providing
-// access to the individual databases. One database per model and an additional
-// model for a global database.
-//
-//	┌─────────────────┐
-//	│                 │
-//	│                 │
-//	│  Controller DB  │
-//	│                 │
-//	│                 │
-//	└─────────────────┘
-//
-//	┌──────────────┐ ┌──────────────┐ ┌──────────────┐
-//	│              │ │              │ │              │
-//	│              │ │              │ │              │
-//	│  Model 1 DB  │ │  Model 2 DB  │ │  Model N DB  │
-//	│              │ │              │ │              │
-//	│              │ │              │ │              │
-//	└──────────────┘ └──────────────┘ └──────────────┘
-//
-// Each database is wrapped via a TrackedDBWorker, which is responsible for
-// managing the lifecycle of the database connection. If a database connection
-// is temporarily lost, the TrackedDBWorker will attempt to reconnect to the
-// database. If the connection is permanently lost, the TrackedDBWorker will
-// terminate the DBAccessor worker and a new one will be started by the worker
-// manager.
-//
-//	┌───────────────────┐
-//	│                   │
-//	│   Txn Runner      │
-//	│                   ├──────────┐
-//	│  ┌─────────────┐  │          │
-//	│  │             │  │        PING
-//	│  │             │  │          │
-//	│  │  Dqlite DB  ◄──┼──────────┘
-//	│  │             │  │
-//	│  │             │  │
-//	│  └─────────────┘  │
-//	│                   │
-//	└───────────────────┘
-//
-// The DBAccessor is officially the only worker that should be accessing the
-// database directly. All other workers, including the apiserver should be
-// accessing the database via the domain services.
+// Package dbaccessor defines workers that create and operate the Dqlite cluster
+// and the individual Dqlite databases and the connections to them. These
+// databases are then made watchable by the internal/worker/changestream package
```

Comment:
I would not put actions of _consumers_ of this package into its documentation. 

Other workers receiving the output of this worker as an input should document their own behaviour.

Reply:
Would you omit these consumers altogether or merely the detail about what they do?

Reply:
I would omit them altogether for the following reasons:
- This is not exhaustive as to the list of consumers of this dependency. For example, the database upgrader worker uses the output transaction runner _without_ its watcher decoration.
- There is another export of this worker that we haven't covered yet, because it's the diminutive case. We talk here about `DBGetter`, but there is also `DBDeleter`, which doesn't factor the change-stream at all.

Reply:
Makes sense, thanks. I've removed all mention of consumers, both from the dbaccessor and from the changestream.